include/seqan3/core/parallel/detail/reader_writer_manager.hpp

```diff
@@ -18,7 +18,7 @@
 #include <seqan3/std/new>
 
 #include <seqan3/core/parallel/detail/latch.hpp>
-#include <seqan3/core/parallel/detail/spin_delay.hpp>
+#include <seqan3/utility/parallel/detail/spin_delay.hpp>
```

Comment:
include order

Reply:
Doesn't really make sense here, the other files get moved to utillity as well in my next PR.